sdk/test/Services/S3/UnitTests/Custom/UseArnRegionTests.cs

```diff
@@ -68,90 +96,73 @@ public void ExplicitlyEnable()
         [TestCategory("S3")]
         public void EnvironmentVariableEnable()
         {
-            Environment.SetEnvironmentVariable("AWS_S3_USE_ARN_REGION", "true");
-            try
-            {
-                var config = new AmazonS3Config
-                {
-                };
+            Environment.SetEnvironmentVariable(AWS_S3_USE_ARN_REGION_ENVIRONMENT_VARIABLE, "true");
```

Comment:
The reason I removed the `try / finally` is because even if a test fails, the next test will just reset the environment variable before continuing with its logic. But I will add it back to be more defensive. Doesn't hurt.

For example we don't do this in the `DisableS3ExpressSessionAuthTests`.